tests/context_mt5_test.py

```diff
@@ -13,24 +13,24 @@ class ContextMT5Tests(unittest.TestCase):
     
     settings = json.loads("""{
```

Comment:
Since we are no longer using settings in the class tested, we should remove this dictionary.

Reply:
Updated!